async_simple/coro/test/LazyTest.cpp

```diff
@@ -467,7 +467,7 @@ TEST_F(LazyTest, testCollectAllBatched) {
         auto out = co_await std::move(combinedLazy);
 
         CHECK_EXECUTOR(&e1);
-        EXPECT_EQ(task_num, out.size());
+        EXPECT_EQ(static_cast<decltype(out.size())>(task_num), out.size());
```

Comment:
Why not use size_t?

Reply:
https://github.com/alibaba/async_simple/pull/146/commits/795c10d2dbe0a84bd94330009d4f89aba083352f